app/models/guide.rb

```diff
@@ -29,6 +29,10 @@ def self.search(search_terms)
     where(:id => ids).order(order)
   end
 
+  def self.with_state(state)
+    Guide.includes(:latest_edition).where(editions: {state: state})
```

Comment:
Hmm, isn't that going to return Guides that have been published in the past but are now in draft if you search for state: 'published'?